pyart/map/gates_to_grid.py

```diff
@@ -95,6 +95,21 @@ def map_gates_to_grid(
     if len(radars) == 0:
         raise ValueError("Length of radars tuple cannot be zero")
 
+    # set min_radius depending on whether processing ARM radars
+    if min_radius is None:
+        min_radius = 250.0
+        try:
+            if "platform_id" in radars[0].metadata.keys():
+                if np.any(
+                    [
+                        x in radars[0].metadata["platform_id"].lower()
+                        for x in ["sacr", "sapr"]
+                    ]
+                ):
+                    min_radius = 100.0
+        except:
```

Comment:
Do you know the specific error that would be thrown here? Ex. key not in dict?

Reply:
1. when metadata is not an attribute (can be fixed in the tests module)
2. when no 'platform_id' attribute in metadata

Reply:
So I think having an `AttributeError` check here instead of a blanket except would be useful to ensure we are catching the right warnings.

Reply:
Makes sense, @mgrover1. Done.